### PR TITLE
check optional measurement cols in mapping

### PR DESCRIPTION
**Pull request by yannikschaelte**

### CODE REVIEW

File: petab/parameter_mapping.py [hunk 1]
```diff
@@ -281,12 +281,12 @@ def _apply_output_parameter_overrides(
...
         # we trust that the number of overrides matches (see above)
         overrides = measurements.split_parameter_replacement_list(
-            row.observableParameters)
+            row.get(observableParameters, None))
         _apply_overrides_for_observable(mapping, row[OBSERVABLE_ID],
                                         'observable', overrides)
 
         overrides = measurements.split_parameter_replacement_list(
-            row.noiseParameters)
+            row.get(noiseParameters, None))
```
Comment (dweindl) [inline petab/parameter_mapping.py]: Codacy Issue found: [Undefined variable 'noiseParameters'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5089700)

File: petab/parameter_mapping.py [hunk 2]
```diff
@@ -281,12 +281,12 @@ def _apply_output_parameter_overrides(
     for _, row in cur_measurement_df.iterrows():
         # we trust that the number of overrides matches (see above)
         overrides = measurements.split_parameter_replacement_list(
-            row.observableParameters)
+            row.get(observableParameters, None))
```
Comment (dweindl) [inline petab/parameter_mapping.py]: Codacy Issue found: [Undefined variable 'observableParameters'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5089700)

File: petab/parameter_mapping.py [hunk 3]
```diff
@@ -359,7 +359,7 @@ def _apply_parameter_table(par_mapping: ParMappingDict,
     if parameter_df is None:
         return
 
-    for row in parameter_df.itertuples():
+    for row in parameter_of.itertuples():
```
Comment (dweindl) [inline petab/parameter_mapping.py]: Codacy Issue found: [Undefined variable 'parameter_of'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5089700)

Comment (dweindl) [reply petab/parameter_mapping.py]: hmmm....

File: petab/parameter_mapping.py (hunk 2 above)
Comment (dweindl) [inline petab/parameter_mapping.py]:
```suggestion
            row.get(OBSERVABLE_PARAMETERS, None))
```

File: petab/parameter_mapping.py (hunk 1 above)
Comment (dweindl) [inline petab/parameter_mapping.py]:
```suggestion
            row.get(NOISE_PARAMETERS, None))
```

**Comment by dweindl** [comment]
Could you please change that for all other calls to `split_parameter_replacement_list`? That would fix #325 as well.

**Comment by yannikschaelte** [comment]
@dweindl hope all occurrences fixed now

**Pull request merged**